web-server/pages/api/internal/[org_id]/settings.ts

```diff
@@ -0,0 +1,48 @@
+import * as yup from 'yup';
+
+import { handleRequest } from '@/api-helpers/axios';
+import { Endpoint } from '@/api-helpers/global';
+import { OrgSettingsApiResponse } from '@/types/resources';
+
+const pathSchema = yup.object().shape({
+  org_id: yup.string().uuid().required()
+});
+
+const getSchema = yup.object().shape({
+  setting_type: yup.string().required()
+});
+
+const putSchema = yup.object().shape({
+  setting_type: yup.string().required(),
+  user_id: yup.string().uuid().optional(),
+  setting_data: yup.object().required()
+});
+
+const endpoint = new Endpoint(pathSchema);
+
+endpoint.handle.PUT(putSchema, async (req, res) => {
+  const { org_id, setting_type, user_id, setting_data } = req.payload;
```

Comment:
`user_id` isn't needed here, or in the schema since we don't have the concept of "Users" in this app yet.